lib/task_helpers/exports.rb

```diff
@@ -0,0 +1,35 @@
+module TaskHelpers
+  class Exports
+    def self.new_filename(filename, keep_spaces = false)
+      new_filename = filename
```

Comment:
I had that line because the gsub returned nil if keep_spaces was true. I've changed the code to eliminate this assignment by using a ternary operator.